public_suffix_list.dat

```diff
@@ -11859,6 +11859,7 @@ gitlab.io
 // Gitplac.si - https://gitplac.si
 // Submitted by Aljaž Starc <me@aljaxus.eu>
 gitpage.si
+gitapp.si
```

Comment:
move above gitpage.si so that the gitplac section domains are alphebetic order by sld